contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaErrorEmailSender.java

```diff
@@ -0,0 +1,301 @@
+package com.netflix.conductor.contribs.kafka;
```

Comment:
Understand the reason for this. But this doesn't belong in Conductor. If an email needs to be sent when event processing fails, that should be handled externally. For instance, a listener on the "errorQueue" that KafkaObservableQueue.processFailures posts to could send that email.

Reply:
Got rid of the email functionality and replaced it with a protected method that can be overridden to provide this or other functionality.